packages/admin/src/routes/dashboard/components/Dashboard/Dashboard.js

```diff
@@ -10,52 +9,44 @@ const StyledDashBoardWrapper = styled.div`
   width: 100%;
   height: 100%;
   background-color: ${theme.color('backgroundBody')};
-`
-
-const InnerWrapper = styled.div`
-  width: 600px;
-  height: 50px;
-  position: relative;
-  left: -320px;
-`
-
-const StyledDashBoard = styled.div`
   display: flex;
-  height: 75%;
   justify-content: flex-end;
   align-items: flex-end;
-  margin-right: -180px;
 
-  ${StyledH1} {
-    font-size: 10rem;
-    letter-spacing: .5rem;
-    color: ${theme.color('primary')};
+  @media (max-width: 568px) {
+    justify-content: center;
+    align-items: center;
   }
+`
 
-  ${InnerWrapper} {
-    position: relative;
-    top: 2rem;
+const StyledSlogan = styled.div`
+  max-width: 900px;
+  width: 100%;
+  height: 100px;
+  text-align: right;
+  margin-bottom: 150px;
+  margin-right: 100px;
+  margin-left: 100px;
+
+  @media (max-width: 568px) {
+    margin-right: 25px;
+    margin-left: 25px;
+    margin-bottom: 0;
+  }
 
-    ${/* sc-selector */StyledH3} {
-      font-size: 3rem;
-      color: ${theme.color('primary')};
-    }
-    text-align: right;
+  ${/* sc-selector */StyledH3} {
```

Comment:
Wouldn't it be better to declare a own styled component for the title 

```
const StyledTitle = styled(StyledH3) {
  font-size: 3rem;
  color: ${theme.color('primary')};	 
}
```

What's your opinion on this?